### ISSUE TITLE
Webpack production build issue

### ISSUE THREAD

**Issue by kbiedrzycki** [ruanmartinelli/is-container#3]
Hey,

I was trying to include package in our application where we use webpack as bundler, however, code exposed by index.js file cannot be minified.

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	../node_modules/is-container/index.js:1 

Read more here: http://bit.ly/2tRViJ9
```

Can you provide minified ES5 version code as well?

**Comment by ruanmartinelli** [comment]
Sure, I'll take a look into it. Thanks for reporting!

**Comment by ruanmartinelli** [comment]
I've just released `1.1.1` which hopefully fixes this issue. Can you test it and tell me if it works for you? :grin:

**Comment by kbiedrzycki** [comment]
Hey,

Yeah, now we can build stuff. However, I think you've introduced breaking change cause now people will have to use `import` statement instead of `require`. Maybe you have to play with `UMD` bundle type to get this working for both?

**Comment by kbiedrzycki** [comment]
Like this -> https://babeljs.io/docs/en/babel-plugin-transform-es2015-modules-umd/

**Comment by kbiedrzycki** [comment]
@ruanmartinelli any update on this?

**Comment by ruanmartinelli** [comment]
Thanks for pointing that out @kbiedrzycki, I was able to rollback it on time. :+1: 

Just released `1.1.2` with the UMD build. Let me know if your issue still persists.

**Comment by kbiedrzycki** [comment]
@ruanmartinelli unfortunately there was still problem with webpack consuming this package, however I've created fix for this here https://github.com/ruanmartinelli/is-container/pull/4

**Comment by ruanmartinelli** [comment]
HOORAY!

Thanks for the PR @kbiedrzycki, glad it finally worked. Just published `1.1.3` :smile:
